pkg/options/describe_test.go

```diff
@@ -89,12 +89,6 @@ func TestDescribeOptions_Ask(t *testing.T) {
 		"sample-task-run3 started 3 minutes ago",
 	}
 
-	options6 := []string{
-		"clustertask1",
-		"clustertask2",
-		"clustertask3",
-	}
-
 	options7 := []string{
```

Comment:
Please update the numbered the test cases so they are in order, or rename them to descriptive instead of numbered, that way we don't have `option4, option5, option7, option8,...` 😅